signer/tests/integration/transaction_coordinator.rs

```diff
@@ -4345,6 +4457,10 @@ async fn coordinator_skip_onchain_completed_deposits(deposit_completed: bool) {
     let start_flag = Arc::new(AtomicBool::new(false));
     let flag = start_flag.clone();
 
+    // We will use network messages to detect the coordinator attempt, so we
+    // need to connect to the network
+    let fake_ctx = ctx.clone();
+
```

Comment:
nit: is there a reason for this move? Its previous location seems more fitting

Reply:
[52c4f73](https://github.com/stacks-sbtc/sbtc/pull/1602/commits/52c4f73f473e673eca89b107b190a50527f42e81)